cluster-autoscaler/cloudprovider/mcm/mcm_manager.go

```diff
@@ -109,10 +111,12 @@ type McmManager struct {
 }
 
 type instanceType struct {
-	InstanceType string
-	VCPU         resource.Quantity
-	Memory       resource.Quantity
-	GPU          resource.Quantity
+	InstanceType     string
+	VCPU             resource.Quantity
+	Memory           resource.Quantity
+	GPU              resource.Quantity
+	EphemeralStorage resource.Quantity
+	NumberOfPods     resource.Quantity
```

Comment:
`PodCount` could be a better variable name. WDYT?

Reply:
sounds good to me